### PR TITLE
[script.module.trakt@matrix] 4.1.0+matrix.1

### PR DESCRIPTION
**Pull request by Razzeee**
### Add-on details:

- **General**
  - Add-on name: trakt.py
  - Add-on ID: script.module.trakt
  - Version number: 4.1.0+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/Razzeee/script.module.trakt
  
Packed for Kodi from https://github.com/fuzeman/trakt.py

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.module.trakt/addon.xml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+    <import addon="script.module.requests" version="2.15.1"/>
+    <import addon="script.module.six" version="1.10.0"/>
+    <import addon="script.module.arrow" version="0.10.0"/>
   </requires>
   <extension point="xbmc.python.module"
              library="lib" />
   <extension point="xbmc.addon.metadata">
-    <summary lang="en_GB">Python interface for the trakt.tv API</summary>
-    <description lang="en_GB">Packed for Kodi from https://github.com/fuzeman/trakt.py</description>
+    <summary lang="en">Python interface for the trakt.tv API</summary>
```
Comment (enen92) [inline script.module.trakt/addon.xml:15]: en_GB

File: script.module.trakt/addon.xml [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+    <description lang="en">Packed for Kodi from https://github.com/fuzeman/trakt.py</description>
```
Comment (enen92) [inline script.module.trakt/addon.xml:16]: en_GB

File: script.module.trakt/addon.xml [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
     <platform>all</platform>
+    <language></language>
```
Comment (enen92) [inline script.module.trakt/addon.xml:18]: drop if empty

File: script.module.trakt/addon.xml [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
     <license>MIT</license>
+    <forum></forum>
```
Comment (enen92) [inline script.module.trakt/addon.xml:20]: drop if empty

File: script.module.trakt/addon.xml [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
     <website>https://github.com/fuzeman/trakt.py</website>
     <source>https://github.com/Razzeee/script.module.trakt</source>
-    <assets>
```
Comment (enen92) [inline script.module.trakt/addon.xml:21]: icon has to be defined in addon.xml

File: script.module.trakt/addon.xml [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
-      <icon>icon.png</icon>
-    </assets>
+    <email></email>
```
Comment (enen92) [inline script.module.trakt/addon.xml:23]: drop if empty

**Comment by Razzeee** [comment]
> 
> 
> Code looks fine, some regressions in the addon manifest. Maybe you didn't push the last update to your upstream repo?

no, just haven't changed it. we changed it for the tvdb module

**Comment by enen92** [comment]
Ah, this was maybe done as part of our bulk script.module migration to matrix to help devs. We fixed a bunch of stuff from the addon.xml of several script.modules

**Comment by enen92** [comment]
See: https://github.com/xbmc/repo-scripts/pull/1413

**Comment by Razzeee** [comment]
Should be fine now

**Pull request merged**